flow360/cloud/requests.py

```diff
@@ -133,3 +133,20 @@ class NewGeometryRequest(Flow360Requests):
         alias="lengthUnit", description="project length unit"
     )
     description: str = pd.Field(default="", description="project description")
+
+
+class NewVolumeMeshRequestV2(Flow360Requests):
+    """[Simulation V2] Creates new project and a new volume mesh resource."""
+
+    name: str = pd.Field(description="project name")
+    solver_version: str = pd.Field(
+        alias="solverVersion", description="solver version used for the project"
+    )
+    tags: List[str] = pd.Field(default=[], description="project tags")
+    file_name: str = pd.Field(alias="fileName", description="file name of the volume mesh")
+    parent_folder_id: str = pd.Field(alias="parentFolderId", default="ROOT.FLOW360")
+    length_unit: Literal["m", "mm", "cm", "inch", "ft"] = pd.Field(
+        alias="lengthUnit", description="project length unit"
+    )
+    description: str = pd.Field(default="", description="project description")
+    format: Union[Literal["cgns"], Literal["aflr3"]] = pd.Field(description="data format")
```

Comment:
@JunchengXue can you review the request format?

Reply:
```
{
  "name": "string",
  "solverVersion": "string",
  "tags": [
    "string"
  ],
  "parentFolderId": "string",
  "lengthUnit": "string",
  "endianness": "string",
  "format": "string",
  "description": "string",
  "fileName": "string",
  "icon": "string"
}
```

Based off Swagger docs for this request. (Endianness is inferred from file and not supported from what I've seen for now and icon is frontend-only)

https://flow360-api.dev-simulation.cloud/swagger-ui/index.html#/flow-360-volume-mesh-controller-v-2/create